src/consumption.cpp

```diff
@@ -967,17 +967,17 @@ void player::modify_health( const islot_comestible &comest )
 void player::modify_stimulation( const islot_comestible &comest )
 {
     if( comest.stim != 0 &&
-        ( abs( stim ) < ( abs( comest.stim ) * 3 ) ||
-          sgn( stim ) != sgn( comest.stim ) ) ) {
+        ( abs( get_stim() ) < ( abs( comest.stim ) * 3 ) ||
```

Comment:
you call get_stim() a bunch here, why not store the value in a const int at the top and then use that?

Reply:
Sure